customer.go

```diff
@@ -96,26 +105,29 @@ type CustomerListParams struct {
 // Customer is the resource representing a Stripe customer.
 // For more details see https://stripe.com/docs/api#customers.
 type Customer struct {
-	AccountBalance      int64                        `json:"account_balance"`
-	Address             Address                      `json:"address"`
-	Created             int64                        `json:"created"`
-	Currency            Currency                     `json:"currency"`
-	DefaultSource       *PaymentSource               `json:"default_source"`
-	Deleted             bool                         `json:"deleted"`
-	Delinquent          bool                         `json:"delinquent"`
-	Description         string                       `json:"description"`
-	Discount            *Discount                    `json:"discount"`
-	Email               string                       `json:"email"`
-	ID                  string                       `json:"id"`
-	InvoicePrefix       string                       `json:"invoice_prefix"`
-	Livemode            bool                         `json:"livemode"`
-	Metadata            map[string]string            `json:"metadata"`
-	Name                string                       `json:"name"`
-	Phone               string                       `json:"phone"`
-	PreferredLocales    []string                     `json:"preferred_locales"`
-	Shipping            *CustomerShippingDetails     `json:"shipping"`
-	Sources             *SourceList                  `json:"sources"`
-	Subscriptions       *SubscriptionList            `json:"subscriptions"`
+	AccountBalance   int64                    `json:"account_balance"`
+	Address          Address                  `json:"address"`
+	Created          int64                    `json:"created"`
+	Currency         Currency                 `json:"currency"`
+	DefaultSource    *PaymentSource           `json:"default_source"`
+	Deleted          bool                     `json:"deleted"`
+	Delinquent       bool                     `json:"delinquent"`
+	Description      string                   `json:"description"`
+	Discount         *Discount                `json:"discount"`
+	Email            string                   `json:"email"`
+	ID               string                   `json:"id"`
+	InvoicePrefix    string                   `json:"invoice_prefix"`
+	Livemode         bool                     `json:"livemode"`
+	Metadata         map[string]string        `json:"metadata"`
+	Name             string                   `json:"name"`
+	Phone            string                   `json:"phone"`
+	PreferredLocales []string                 `json:"preferred_locales"`
+	Shipping         *CustomerShippingDetails `json:"shipping"`
+	Sources          *SourceList              `json:"sources"`
+	Subscriptions    *SubscriptionList        `json:"subscriptions"`
+	TaxIds           *TaxIDList               `json:"tax_ids"`
```

Comment:
Let's do `TaxIDs` here instead.

Reply:
ah nice catch, surprised the linter did not say anything, it complained about a lot of other fields! Fixed, PTAL